plugin/fulltext/mysql-test/ngram/ngram_ucs2.test

```diff
@@ -0,0 +1,15 @@
+# Encoding with mbminlen >= 2.
+
+SET NAMES utf8mb4;
+
+--error 1283
+CREATE TABLE t (str TEXT, FULLTEXT INDEX (str) WITH PARSER ngram) Engine=InnoDB CHARSET=UCS2;
```

Comment:
I do not think that `SET NAMES` is necessary in this test, because the file looks like ASCII only.

Please avoid numeric `error` codes. You can find the symbolic code of 1283 in `include/mysqld_ername.h` in the build directory.

Reply:
Removed `SET NAMES` from the file, and changed numeric codes to symbolic name. (In this case it was `ER_BAD_FT_COLUMN`).